Corona-Warn-App/src/main/java/de/rki/coronawarnapp/SecurityProvider.kt

```diff
@@ -0,0 +1,21 @@
+package de.rki.coronawarnapp
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+import org.conscrypt.Conscrypt
+import timber.log.Timber
+import java.security.Security
+import javax.inject.Inject
+
+class SecurityProvider @Inject constructor() {
+
+    fun setup() {
+        try {
+            Timber.d("Setting up security provider")
+            // Enable Conscrypt for TLS1.3 Support below API Level 29
+            Security.insertProviderAt(Conscrypt.newProvider(), 1)
+            Security.addProvider(BouncyCastleProvider()) // For SHA256withRSA/PSS
```

Comment:
Could you add a class name or PR comment here for what this is used for?
And a comment to the class using it too, so that when either is removed/changed, the new dev knows what to update or check?